Libraries/LibGUI/TextEditor.cpp

```diff
@@ -1406,13 +1406,28 @@ void TextEditor::recompute_visual_lines(size_t line_index)
     if (is_line_wrapping_enabled()) {
         int line_width_so_far = 0;
 
+        size_t last_whitespace_index = 0;
+        size_t line_width_since_last_whitespace = 0;
         auto glyph_spacing = font().glyph_spacing();
         for (size_t i = 0; i < line.length(); ++i) {
             auto code_point = line.code_points()[i];
+            if (code_point == 32) {
```

Comment:
Let's compare against `' '` instead of `32`.

Reply:
There's probably a bunch of whole bunch of whitespace characters which should be checked.

Space `0x20` and tab `0x09` are the most immediately obvious.

Reply:
why not just use `isspace()` then?